docs/model-hub-library/mmdetection/_index.rst

```diff
@@ -12,9 +12,8 @@ Faster-RCNN and Mask-RCNN in addition to cutting edge methods from the research
 
 **model-hub** makes it easy to use MMDetection with Determined while keeping the developer
 experience as close as possible to what it's like working directly with **MMDetection**. Our library
-serves as an alternative to the trainer used by MMDetection (see `mmcv's runner
-<https://mmcv.readthedocs.io/en/latest/understand_mmcv/runner.html>`_) and provides access to all of
```

Comment:
Use `https://mmcv.readthedocs.io/en/v1.7.0/understand_mmcv/runner.html`. We use an outdated version of `mmcv` in our repo, where this object still applies. This link should only be removed when we decide to change `model_hub`.